creator-node/src/routes/files.js

```diff
@@ -147,7 +147,7 @@ const getStoragePathQueryCacheKey = (path) => `storagePathQuery:${path}`
 
 const logGetCIDDecisionTree = (decisionTree, req) => {
   try {
-    req.logger.info(`[getCID] Decision Tree: ${JSON.stringify(decisionTree)}`)
+    req.logger.debug(`[getCID] Decision Tree: ${JSON.stringify(decisionTree)}`)
```

Comment:
this is by far the largest win as far as loggly is concerned. basically we don't send these logs to loggly but they'll be around for local dev